src/lib/Vault.ts

```diff
@@ -1,30 +1,48 @@
 import fs from 'fs'
+import tls from 'tls'
+import net from 'net'
 import Path from 'path'
 import hkdf from 'futoin-hkdf'
+import git from 'isomorphic-git'
 import { EncryptedFS } from 'encryptedfs'
+import { Address } from '@polykey/peer-store/PeerInfo'
+import { efsCallbackWrapper } from '@polykey/utils'
+import httpRequest from './httpRequest'
+
+
+type VaultMetadata = {
+  sharedPubKeys: string[]
+}
 
 class Vault {
 
   private key: Buffer
   private keyLen: number
   name: string
-  private fs: EncryptedFS
+  efs: EncryptedFS
```

Comment:
Are these meant to be private properties?

Reply:
this should be accessible in some way for the `GitServer` but they definitely shouldn't be settable. I have changed this to private with a public getter :+1: